From: Tim Harvey <tharvey@gateworks.com>
Date: Thu, 5 May 2022 12:50:10 -0700
Subject: Bluetooth: btbcm: Add entry for BCM4373A0 UART Bluetooth
Patch-mainline: v5.19-rc1
Git-commit: 0d37ddfc50d9a81f46d3b2ffa156d4fea8a410f7
References: jsc#PED-1407

This patch adds the device ID for the BCM4373A0 module, found e.g. in
the Infineon (Cypress) CYW4373E chip. The required firmware file is
named 'BCM4373A0.hcd'.

Signed-off-by: Tim Harvey <tharvey@gateworks.com>
Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/bluetooth/btbcm.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/bluetooth/btbcm.c
+++ b/drivers/bluetooth/btbcm.c
@@ -458,6 +458,7 @@ static const struct bcm_subver_table bcm
 	{ 0x6106, "BCM4359C0"	},	/* 003.001.006 */
 	{ 0x4106, "BCM4335A0"	},	/* 002.001.006 */
 	{ 0x410c, "BCM43430B0"	},	/* 002.001.012 */
+	{ 0x2119, "BCM4373A0"	},	/* 001.001.025 */
 	{ }
 };
 
